cockroachcloud/monitoring-page.md

```diff
@@ -72,6 +72,22 @@ The [available metrics](#available-metrics) are drawn directly from the Cockroac
 Metric values and time-series graphs in Datadog are not guaranteed to match those in the [DB Console](#access-the-db-console), due to differences in how CockroachDB and Datadog calculate and display metrics.
 {{site.data.alerts.end}}
 
+#### Enable percentiles for selected metrics
+
+A subset of CockroachDB metrics require that you explicitly enable percentiles for them in the Datadog interface, or graphs that display data for them will fail to render properly.
```

Comment:
```suggestion
A subset of CockroachDB metrics require that you explicitly enable percentiles for them in the Datadog interface. Graphs that display data for these metrics will fail to render properly otherwise.
```